doCNA/Distribution.py

```diff
@@ -120,7 +149,11 @@ def fit_double_G (values_all, alpha, r = 0.5, initial_bounds = None, initial_p0
     Returns dictionary with parameters.
     """
     
-    thr0 = get_outliers_thrdist (np.sort(values_all), alpha, r)   
+    try:
```

Comment:
when does this happen? why would this raise an exception?

Reply:
same as above